spec/support/fixtures/foo

```diff
@@ -30,10 +30,10 @@ module Foo
       end
     end
 
-    register 'hello',     Hello
-    register 'version',   Version
+    register 'hello', Hello
+    register 'version', '-v', '--version', Version
 
-    register 'generate model',  Generate::Model
+    register 'generate model', 'generate --model', Generate::Model
```

Comment:
@AlfonsoUceda Can we keep this realistic? It's probably not the case that someone will define a command like this. What if we leave the line as it is, but instead we add:

```ruby
register 'generate --help', Generate::Help
```